features/account/customer_account/viewing_orders_history/order_is_always_placed_in_a_base_currency_of_a_channel.feature

```diff
@@ -0,0 +1,30 @@
+@customer_account
+Feature: Order is always placed in a base currency of a channel
+    In order to pay exact amount of money
+    As an Customer
+    I want to be charged in a base currency of channel
+
+    Background:
+        Given the store operates on a channel named "United States" in "USD" currency
+        And that channel allows to shop using the "CAD" currency with exchange rate 3.0
+        And the store ships to "United States"
+        And the store has a zone "United States" with code "US"
+        And this zone has the "United States" country member
```

Comment:
We should use `DefaultUnitedStatesChannelFactory` and then the 3 steps above won't be necessary.